cmd/release/docs/internal/template_writer.go

```diff
@@ -0,0 +1,128 @@
+package internal
+
+import (
+	. "../../internal"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"log"
+	"os"
+	"text/template"
+)
+
+type Doc struct {
+	Filename     string
+	TemplateName string
+	IsIncluded   bool
+	AppendToEnd  func(release *Release) (string, error)
+}
+
+var closeFile = func(f *os.File) {
```

Comment:
If you don't need to set this value later, you can define this as a normal function `func closeFile(f *os.File)`
Not a big deal though